apps/hubble-stats/app/page.tsx

```diff
@@ -1,9 +1,12 @@
-import { KeyMetricsTable } from '../components';
+'use client';
```

Comment:
Could you please explain why this component is designated as a client component?

It might be beneficial to use a server component whenever possible, as it could help reduce the JS code on the client side.